curbmap-ios-mvp/Model/User.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  User.swift
+//  curbmap
+//
+//  Copyright © 2017 curbmap. All rights reserved.
+//
+
+import Foundation
+import MapKit
+
+let currentUser = User(username: "curbmap", password: "TestCurbm@p1");
```

Comment:
Instead of using global, can we do the singleton as a type property within the User class?  That way we can eventually have type methods for the currentUser, which could be very useful down the road.  

Also I recommend us implement logic to instantiate currentUser through login methods, even for testing purpose.  This is because we could test login function in various networking situations.